src/constants/productTokens.ts

```diff
@@ -3,11 +3,13 @@ import bedBorderLogo from 'assets/bed-border.png'
 import dataLogo from 'assets/data-logo.png'
 import indexLogo from 'assets/index-token.png'
 
-export const dpiTokenImage = 'https://index-dao.s3.amazonaws.com/defi_pulse_index_set.svg'
+export const dpiTokenImage =
```

Comment:
Should we have a new product entry for ETH2x-FLI polygon? Its different from DPI/MVI polygon because it's not bridged right?